### ISSUE TITLE
Colon rule not triggered when badly formatted assignment operator is used in declaration statement

### ISSUE THREAD

**Issue by mgrebenets** [realm/SwiftLint#135]
Colon rule is not triggered if assignment operator is used in declaration statement and that assignment operator is badly formatted, e.g. no spaces around `=`.

```swift
// This triggers colon rule
let x :Int = 0
let x2:Int = 0

// This does not trigger colon rule
let y :Int=0
let y2:Int=0
```

**Comment by tom-sparo** [comment]
Are you sure?

<img width="709" alt="screen shot 2015-10-17 at 10 45 40 am" src="https://cloud.githubusercontent.com/assets/12956264/10559665/4574db7a-74bc-11e5-8d69-66a5c7344e78.png">

**Comment by mmorier** [comment]
Fixed in master. Reopen if you stil have the problem.
